### ISSUE TITLE
Examples for pf/freebsd

### ISSUE THREAD

**Issue by PaulGWebster** [darkk/redsocks#101]
Has anyone managed to get redsocks working with pf, if so please post your firewall rdr's :)

especially for Socks5/UDP

**Comment by darkk** [comment]
Yep, I'd also like some example as that's basically the question I need to answer before fixing #110 :)

**Comment by nomadrick** [comment]
Has anyone been able to compile redsocks in freebsd?  If so, please point me in to the how.

**Comment by PaulGWebster** [comment]
Fun when you google a question and find you asked it a year previous :)
